lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,22 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    // Set digital object URL
+    $digitalObjects = $this->getDigitalObjects();
```

Comment:
Does this return only the master digital object (information_object.id = digital_object.information_object_id)?  IIRC it doesn't return derivatives.